jenkins/config_backup/config_backup_playbook.yml

```diff
@@ -34,6 +35,16 @@
         dest: "{{ config_backup_dir }}/tiaas/"
       with_items:
         - db.sqlite3
+    - name: Copy list of venv packages
+      block:
+      - name: Get list of packages
+        command: /mnt/galaxy/venv/bin/pip list --format=freeze
```

Comment:
@cat-bro would `command: /mnt/galaxy/venv/bin/pip freeze` be more suitable for reproducibility?

Reply:
yes indeed! thank you